### PR TITLE
thClass implementation + typo fix

### PR DESCRIPTION
**Pull request by AZeghers**
thClass is now properly implemented on KtTableColumn

### CODE REVIEW

File: docs/pages/components/tables.vue [hunk 1]
```diff
@@ -584,7 +584,7 @@ slot-scope is not required for the `loading` and `empty` slots
 | `hiddenColumns`          | prop for setting hidden columns                                     | `Array`                     | [{ prop, hidden }]    | `[]`    |
 | `filterdColumns`         | prop for setting filterd columns                                    | `Array`                     | [{ prop, filter }]    | `[]`    |
 | `tdClasses`              | classes to apply to all `<td>` elements                             | `Array`, `String`, `Object` | `"responsive"`        | `[]`    |
-| `thClass`                | classes to apply to all `<th>` elements                             | `Array`, `String`, `Object` | `"responsive"`        | `[]`    |
```
Comment (amrdraz) [inline docs/pages/components/tables.vue]: in addition to thClasses there's also thClass which applies to only the column not all columns
so maybe we should asdd that in the docs

also not it's always possible to target `.kt-table  th[prop="propname"]`

Comment (AZeghers) [reply docs/pages/components/tables.vue]: thClass already exists in the column docs, it was just not implemented

This change fixes the fact that thClasses in the KtTable attributes was wrongfully called thClass

Comment (amrdraz) [reply docs/pages/components/tables.vue]: oh guess I missed it due to the local scope of the comment ok all clear then

**Comment by FlorianWendelborn** [comment]
fix #75

**Pull request merged**